src/ansys/pytwin/evaluate/tbrom.py

```diff
@@ -200,7 +264,7 @@ def _read_points(self, filepath):
         return has_point_file
 
     def _init_pointsdata(self, filepath):
-        self._outbasis = TbRom._read_basis(filepath)
+        self._outbasis = TbRom._read_basis(filepath).reshape(self.nb_modes, self.nb_points, self.field_output_dim)
```

Comment:
1st comment - there is no particular error here; the idea is that have a 3d shape array (nb_modes x nb_points x field_output_dim) so that this can be easily used to define the pyvista object used for projection (in particular when interpolation is required - pointsdata[str(i)] = self._outbasis[i]) as well as when updating field data (update_output_field)

2nd comment - I had considered that aspect as well but the _read_basis is also used in other context (input field projection) for which those additional dimensions (in particular field dimensions) are not relevant for now